internal/encoding/gotypes/generate.go

```diff
@@ -772,3 +794,32 @@ func emitDocs(printf func(string, ...any), name string, groups []*ast.CommentGro
 		}
 	}
 }
+
+// gatherImportAliases collects the aliases from imports across the instance.
+// We explicitly ignore "_" aliases because CUE does not support.
```

Comment:
CUE source code should never be using underscore-aliased imports, so I would delete this line as well as the check for `_` below - it just shouldn't happen